lang/cs/Org.Apache.REEF.Tests/Network/GroupCommunicationTests.cs

```diff
@@ -696,8 +696,7 @@ public static List<ICommunicationGroupClient> CommGroupClients(string groupName,
             IPEndPoint nameServerEndpoint, IObserver<NsMessage<GroupCommunicationMessage>> handler)
         {
             return new NetworkService<GroupCommunicationMessage>(
-                0, nameServerEndpoint.Address.ToString(), nameServerEndpoint.Port, 
-                handler, new StringIdentifierFactory(), new GroupCommunicationMessageCodec());
+                0, handler, new StringIdentifierFactory(), new GroupCommunicationMessageCodec(), new NameClient(nameServerEndpoint.Address.ToString(), nameServerEndpoint.Port));
```

Comment:
This test didn't use injection to get NetworkService instance. If it was, yes, we don't need to change as NameClient will be automatically injected given port and address. 
Another option is to change this test case to use injection. It seems there are more to clean for old test cases :).